APIManager.js

```diff
@@ -0,0 +1,87 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+    constructor() {
+        this.data = {
+            mainUser: {},
```

Comment:
you could just initialize this as an empty object. You should be able to delete all these keys and it still work.